src/core/map_printer.cpp

```diff
@@ -1235,8 +1237,8 @@ bool MapPrinter::printMap(QPrinter* printer)
 	int step = 0;
 	auto num_steps = v_page_pos.size() * h_page_pos.size();
 	const QString message_template( (options.mode == MapPrinterOptions::Separations) ?
-	  tr("Processing separations of page %1...") :
-	  tr("Processing page %1...") );
+	  ::OpenOrienteering::MapPrinter::tr("Processing separations of page %1...") :
+	  ::OpenOrienteering::MapPrinter::tr("Processing page %1...") );
```

Comment:
That's part of "exposes Mapper to all namespace-related issues of lupdate." Without this, `lupdate` would put the translation in the wrong context. At the moment I'm not much inclined to fix issues in `lupdate`...

Reply:
Thanks for the explanation. Then the `lupdate` limitation should be documented either in the commit message or a comment so that future Mapper developers can decide when to remove the workaround.